app/components/ChartSettings/index.js

```diff
@@ -21,14 +22,25 @@ class ChartSettings extends Component {
     return -1 !== this.state.modules.indexOf(name);
   }
 
+  _toggleModuleCallback(expanded, name) {
+    if (expanded) {
+      this.setState({
+        activeSettings: name,
```

Comment:
The "settings" are referenced as modules everywhere else. For consistency I would do that here as well.

Also, "expanded" over active.

`expandedModule`